scripts/qa_crowbarsetup.sh

```diff
@@ -4320,8 +4325,14 @@ function ping_fips
 # Use $heat_stack_params to provide parameters to heat template
 function oncontroller_testpreupgrade
 {
-    heat --insecure stack-create upgrade_test -f /root/scripts/heat/2-instances-cinder.yaml $heat_stack_params
-    wait_for 15 20 "heat --insecure stack-list | grep upgrade_test | grep CREATE_COMPLETE" \
+    create_cmd="heat --insecure stack-create upgrade_test -f "
+    list_cmd="heat --insecure stack-list"
+    if iscloudver 7plus; then
+        create_cmd="openstack --insecure stack create upgrade_test -t "
+        list_cmd="openstack --insecure stack list"
```

Comment:
Just wondering: doesn't the `openstack stack` command work with SOC6 already?

Reply:
It does not, but I can deploy SOC6 and investigate further if you wish.

Reply:
I guess there's no need if you are sure.